metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -500,3 +321,10 @@ def close(self):
         self.prepare_for_commit()
         if self.consumer:
             self.consumer.close()
+
+
+class SchemaRegistry:
```

Comment:
1. Rename to KafkaSchemaRegistryBase
2. Extract this into a separate file so that the dependencies are clean for the derived classes.
3. Make it an abstract class - see [this example](https://github.com/acryldata/datahub-fork/blob/master/metadata-ingestion/src/datahub/configuration/common.py#L66)